### ISSUE TITLE
NoStackTraceThrowable exception with quarkus-redis-client

### ISSUE THREAD

**Issue by gcharondkt** [quarkusio/quarkus#31948]
### Describe the bug

Hello,

I'm using `quarkus-redis-client` following [this guide](https://quarkus.io/guides/redis-reference#sentinel-mode), with the properties below :
```properties
quarkus.redis.client-type=sentinel
quarkus.redis.hosts=redis://redis.my-app.svc.cluster.local:26379
quarkus.redis.masterName=redis-master
quarkus.redis.clientName=
quarkus.redis.password=<hidden>
```

Behind the K8S service, I have 3 redis pods, configured with a sentinel mode.

When my app does a GET command, I get the following error : 
> Caused by: io.vertx.core.impl.NoStackTraceThrowable: No more endpoints in chain.

Here is the code throwing an exception : 
```java
    @Inject
    ReactiveRedisDataSource reactiveRedisDataSource;

    private ReactiveValueCommands<String, byte[]> reactiveValueCommands = reactiveRedisDataSource.value(byte[].class);

    public byte[] get(Object key) {
        return reactiveValueCommands.get(key.toString()).await().indefinitely();
    }
```

For the record, it works if I configure my app without the sentinel mode (no `quarkus.redis.client-type` property and port `6379` instead of `26379`). But obviously, I need to make it work with the sentinel mode.

What am I missing ? Is this a bug, or a config issue on my side ?

Thank you for your help.

### How to Reproduce?

https://github.com/gcharondkt/quarkus-redis-sentinel

### Output of `uname -a` or `ver`

Linux ddfs-consumer-59889f55f7-9s46z 5.10.147+ #1 SMP Thu Nov 10 10:14:30 UTC 2022 x86_64 Linux

### Output of `java -version`

openjdk version "19.0.2" 2023-01-17
OpenJDK Runtime Environment Temurin-19.0.2+7 (build 19.0.2+7)
OpenJDK 64-Bit Server VM Temurin-19.0.2+7 (build 19.0.2+7, mixed mode, sharing)

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

Maven home: /opt/homebrew/Cellar/maven/3.8.7/libexec
Java version: 19.0.1, vendor: Homebrew, runtime: /opt/homebrew/Cellar/openjdk/19.0.1/libexec/openjdk.jdk/Contents/Home
Default locale: fr_FR, platform encoding: UTF-8
OS name: "mac os x", version: "13.2", arch: "aarch64", family: "mac"

**Comment by cescoffier** [comment]
Can you provide a reproducer (especially the instruction to run a local redis in sentinel mode)?

**Comment by gcharondkt** [comment]
Shame on me, it was all my fault...


On my reproducer only I've set the property `quarkus.redis.host` instead of `quarkus.redis.hosts`. It works better with the S. Quarkus was using the embedded Redis testcontainer with the 49634 port which was not working in sentinel mode, and I didn't noticed that.
On both the reproducer and the original setup, I've set the property `quarkus.redis.masterName` instead of `quarkus.redis.master-name`


I think the Exception should be more explicit. I didn't get what it means before debugging the code which was throwing it.
What about `No more endpoints in chain : none of the provided endpoints redis://localhost:49634 was able to give the master. ERR unknown command 'sentinel'` ?
